clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -43,6 +43,30 @@ void SYCLInstallationDetector::getSYCLDeviceLibPath(
   DeviceLibPaths.emplace_back(D.SysRoot + "/lib");
 }
 
+void SYCLInstallationDetector::AddSYCLIncludeArgs(
+    const ArgList &DriverArgs, ArgStringList &CC1Args) const {
+  // Add the SYCL header search locations in the specified order.
+  //   ../include/sycl
+  //   ../include/sycl/stl_wrappers
+  //   ../include
```

Comment:
@intel/llvm-reviewers-runtime, I suggest we remove the need for the driver to add 3 different locations for SYCL headers restricted by specific order. Considering that we expose only one header to the user API - `sycl/sycl.hpp`, the driver can include only one location and runtime library project can put everything into a single location or include from other locations. Right?

Reply:
I don't know about `include/sycl` vs `include/`, but the `stl_wrappers` path has to be added in the driver.

Reply:
Are there any conflicts between the files in `../include/sycl/stl_wrappers` and `../include`?

Reply:
I believe maintainability benefits of having these headers in a separate dedicated folder outweigh the costs of having driver add one extra include path, if that's where you're going with your question.

Reply:
What exactly is "maintainability benefits"? Note, these are directories of deployed compiler. We can still have separate directories in the sources code.

Reply:
First, "smart" deploy is extra maintainability burden. Second, I want to be able to remove `stl_wrappers` from search path via manually modifying `clang -cc1`, merging them together with other includes would prevent that. I'm sure there are "third"/"fourth" if I were to think about it just a bit more.